tests/ok_comm/test_ok_comm_barcode.py

```diff
@@ -12,12 +12,12 @@
 import pytest
 from stdlib_utils import invoke_process_run_and_check_errors
 
-from .fixtures import QUEUE_CHECK_TIMEOUT_SECONDS
-from .fixtures_barcode import fixture_test_barcode_simulator
-from .fixtures_ok_comm import fixture_four_board_comm_process
-from .helpers import confirm_queue_is_eventually_empty
-from .helpers import confirm_queue_is_eventually_of_size
-from .helpers import put_object_into_queue_and_raise_error_if_eventually_still_empty
+from ..fixtures import QUEUE_CHECK_TIMEOUT_SECONDS
+from ..fixtures_barcode import fixture_test_barcode_simulator
+from ..fixtures_ok_comm import fixture_four_board_comm_process
```

Comment:
There are a few cases where we make a fixture mainly for certain files (i.e. ok_comm.py) that sometimes get used in another like process_monitor.py. I thought it would be better to leave them where they were with the thought of eventually just making a fixtures folder